server/app/mutations/host_nodes/update.rb

```diff
@@ -4,19 +4,81 @@ module HostNodes
   class Update < Mutations::Command
 
     include Common
+    include Logging
 
     required do
       model :host_node
       array :labels, nils: true
     end
 
+    optional do
+      string :availability, in: ['active', 'drain']
```

Comment:
I think it was `pause` on some other orchestrator. (could not find it though on Docker Swarm or Rancher docs, maybe it was Kube). Anyway that can be added as separate PR IMO.